web_external/util/annotationGeometryParser.js

```diff
@@ -1,154 +1,261 @@
+//=============================================================================
```

Comment:
Could you tell me why you added these separators lines?

Reply:
...because it's *much* easier to skim the code and identify critical logic boundaries (e.g. beginnings of functions) vs. having to read a wall of text. These are SOP in ViViA and fairly common in other vision projects (e.g. KWIVER).

Reply:
Are the other projects JavaScript project? This basically add a line before all class and member without emphasis. I really don't see the different between adding this and a spare line. The class and member function are already identified by their intentation. And they are causing different eslinting errors.

Reply:
They are *much easier* to pick out of the clutter compared to pure whitespace.